index.js

```diff
@@ -57,10 +57,10 @@ exports.init = function (sbot, config) {
   const legacy = _legacy(layered)
 
   // glue modules together
-  if (config.friends && config.friends.hookAuth !== false)
+  if (!config.friends || config.friends.hookAuth !== false)
```

Comment:
I think what you wanted to write here was just
`if (config.friends.hookAuth !== false)`
but you had to protect against `config.friends` being undefined.

The correct way is `if (!config.friends ||` because we want the default to be "run the glue", even if `config.friends` is undefined.

Reply:
Ah, default values. Right. Hmm, I just find these kind of logic expressions hard to read.

Reply:
That is a good find, I was a breaking change before...

Reply:
maybe // default: true comment on this line?

Reply:
Sure

Reply:
Fun fact: [`?.` optional chaining](https://github.com/tc39/proposal-optional-chaining) and [`??` nullish coalescing](https://github.com/tc39/proposal-nullish-coalescing) are already standardized, supported in all green browsers, and supported in Node.js 14:

```js
var a = {}
console.log(a?.foo?.bar ?? 'yo')
// 'yo' 
```

Sometime in the next few years we can start using it to create defaults.